kubernetes/crd/domain-crd.yaml

```diff
@@ -347,6 +347,145 @@ spec:
                     name:
                       type: string
                 type: array
+              fluentdSpecification:
+                description: Automatic fluentd sidecar injection. If specified, the
+                  operator will deploy a sidecar container alongside each WebLogic
+                  Server instance that runs the exporter. WebLogic Server instances
+                  that are already running when the `monitoringExporter` field is
+                  created or deleted, will not be affected until they are restarted.
+                  When any given server is restarted for another reason, such as a
+                  change to the `restartVersion`, then the newly created pod  will
+                  have the fluentd sidecar or not, as appropriate
+                properties:
+                  image:
+                    description: The Fluentd container image name. Defaults to fluent/fluentd-kubernetes-daemonset:v1.14.5-debian-elasticsearch7-1.1
```

Comment:
Can we choose an image that doesn't have "debian" in the tag? It's okay if it's the same content, but I don't want to highlight that this isn't an OL based image.

Reply:
All fluentd images are debian based.  https://hub.docker.com/r/fluent/fluentd-kubernetes-daemonset/